rl_zoo3/push_to_hub.py

```diff
@@ -198,10 +198,9 @@ def package_to_hub(
 
     # Git pull
     repo_local_path = Path(local_repo_path) / repo_name
-    repo = Repository(repo_local_path, clone_from=repo_url)
-    repo.git_pull(rebase=True)
 
-    repo.lfs_track(["*.mp4"])
```

Comment:
no more distinction about what is tracked with git and git lfs?

Reply:
see L203